tests/test_api_capital.py

```diff
@@ -2,12 +2,34 @@
 from walrus.api import get_capital
 
 
-def test_capital_correct():
+def test_capital_single_correct():
     assert get_capital("Australia") == "Canberra"
     assert get_capital("Australia", raise_errors=True) == "Canberra"
 
 
-def test_capital_incorrect():
+def test_capital_tuple_correct():
+    assert get_capital(("Australia", "France")) == {"Australia": "Canberra", "France": "Paris"}
+    assert get_capital(("Australia", "France"), raise_errors=True) == {"Australia": "Canberra", "France": "Paris"}
+
+
+def test_capital_list_correct():
+    assert get_capital(["Australia", "France"]) == {"Australia": "Canberra", "France": "Paris"}
+    assert get_capital(["Australia", "France"], raise_errors=True) == {"Australia": "Canberra", "France": "Paris"}
+
+
+def test_capital_single_incorrect():
     assert get_capital("Austraalia") is None
-    with pytest.raises(ValueError, match=("No data found for country: Austraalia")):
+    with pytest.raises(ValueError, match="No data found for country: Austraalia"):
         get_capital("Austraalia", raise_errors=True)
+
+
+def test_capital_tuple_incorrect():
+    assert get_capital(("Australia", "Fraance")) == {"Australia": "Canberra", "Fraance": None}
+    with pytest.raises(ValueError, match="No data found for country: Austraalia"):
```

Comment:
I'm a bit confused with this check - Australia is spelled correctly while France is misspelled, so shouldn't we have got an Exception about Fraance?

Reply:
ah never mind - now I see the misspellings are flipped in the next line.

Reply:
Those two are two different test cases. Line 27 is the one that has not been passed with the raise_errors argument, while line 28 and 29 deal with the case that has been passed with the raise_errors argument as true.